StaticAnalyzer/views/android/cert_analysis.py

```diff
@@ -54,43 +52,20 @@ def cert_info(app_dir, app_file, tools_dir):
     """Return certificate information."""
     try:
         logger.info('Reading Code Signing Certificate')
-        cert = os.path.join(app_dir, 'META-INF/')
-        cp_path = tools_dir + 'CertPrint.jar'
-        files = [f for f in os.listdir(
-            cert) if os.path.isfile(os.path.join(cert, f))]
-        certfile = None
         dat = ''
         manidat = ''
-        manifestfile = None
-        if 'CERT.RSA' in files:
-            certfile = os.path.join(cert, 'CERT.RSA')
-        else:
-            for file_name in files:
-                if file_name.lower().endswith('.rsa'):
-                    certfile = os.path.join(cert, file_name)
-                elif file_name.lower().endswith('.dsa'):
-                    certfile = os.path.join(cert, file_name)
-        if certfile:
-            args = [settings.JAVA_BINARY, '-jar', cp_path, certfile]
+        issued = ''
+
+        dat = androguard_certinfo(app_dir, app_file)
+
+        if re.findall(r'Apk is signed', dat):
             issued = 'good'
-            try:
-                dat = subprocess.check_output(args)
-                unicode_output = str(dat, encoding='utf-8', errors='replace')
-                dat = escape(unicode_output).replace('\n', '</br>')
-            except Exception:
-                dat = androguard_certinfo(app_dir, app_file)
-        else:
-            dat = 'No Code Signing Certificate Found!'
+        if re.findall(r'Missing certificate', dat):
             issued = 'missing'
```

Comment:
When will this happen. Does that means it's a bad apk that's not properly signed